Sources/Machine/rbm_spin.cc

```diff
@@ -24,6 +24,23 @@ namespace netket {
 
 RbmSpin::RbmSpin(std::shared_ptr<const AbstractHilbert> hilbert, Index nhidden,
                  Index alpha, bool usea, bool useb)
+/* Variables are:
+ * hilbert - the variable containing information about physical system
+ * nhidden - number of neurons in hidden layer defined by larger of the values
+ *           (alpha * number of neuron from visible layer) or nhidden
+ * alpha - density defined as (number of neurons in hidden layer / number of
+ *         neurons in visible layer)
+ * usea - if true use biases in visible layer a*output
+ * useb - if true use biases in hidden layer b*output
+ */ 
+        
+/* Defining variational parameters: weights and biases (a_ for 
+ * visible layer, b_ for hidden layer)
+ *
+ * theta_ correspond to the "output" of a hidden layer 
+ * theta = W_.transpose() * visible + b_. It is used as the input of cosh
+ * functions in the expression for a wavefunction. 
+ */
```

Comment:
This is the source file (.cc), with the method's implementation.
I meant that comments pertaining to the input of the methods should go in the method's declaration in the header file (.h)

Reply:
I see. I'll update it there.